app/models/server.rb

```diff
@@ -1,3 +1,12 @@
+class Scope
+  include Mongoid::Document
+  field :name, type: String
+  field :description, type: String
+  field :selected, type: Boolean, default: false
+  field :elem_id, type: String
+  embedded_in :server
+end
```

Comment:
can we move scope to its own file... it seems like it would be hard to find here